### PR TITLE
add filters to base-filters

### PR DESCRIPTION
**Pull request by jandersson**
The most recent batch of strings that set off status-monitor

### CODE REVIEW

File: data/base-filters.yaml [hunk 1]
```diff
@@ -424,3 +424,14 @@ filter_strings:
 - '^Commenting favorably on the skills you already possess. \w+ gives you some tips on .*$'
 - '^The fire shard glances off .* shield\.$'
 - '^You begin chanting a psalm to invoke the .* spell\.$'
+- '^Holding mana in this room is becoming very difficult\.  Perhaps you should move on\?$'
+- '^No miners appear to be nearby, and you will be unable to deed resources mined in this area\.$'
+- '^An attendant arrives, takes your deed and motions to two burly workers\.'
```
Comment (rpherbig) [inline data/base-filters.yaml]: ❓ Should this line have `$` at the end?

File: data/base-filters.yaml [hunk 2]
```diff
@@ -424,3 +424,14 @@ filter_strings:
...
+- '^The mental strain of initiating a cyclic spell so recently prevents you from formulating the spell pattern\.$'
+- '^A few small orbiting slivers of lunar magic'
+- '^Wouldn.t it be better if you used a melee weapon\?$'
+- '^A .+ collapses as if dead\.$'
+- '^The fluorescent .+ guides your attack\.$'
+- '^Something in the area interferes with your spell preparations\.$'
+- '^Trabe Chalice'
```
Comment (rpherbig) [inline data/base-filters.yaml]: 💡 This is a bit brief, is there a more meaningful message to match?

File: data/base-filters.yaml [hunk 3]
```diff
@@ -424,3 +424,14 @@ filter_strings:
 - '^Commenting favorably on the skills you already possess. \w+ gives you some tips on .*$'
 - '^The fire shard glances off .* shield\.$'
 - '^You begin chanting a psalm to invoke the .* spell\.$'
+- '^Holding mana in this room is becoming very difficult\.  Perhaps you should move on\?$'
+- '^No miners appear to be nearby, and you will be unable to deed resources mined in this area\.$'
+- '^An attendant arrives, takes your deed and motions to two burly workers\.'
+- '^The mental strain of initiating a cyclic spell so recently prevents you from formulating the spell pattern\.$'
+- '^A few small orbiting slivers of lunar magic'
```
Comment (rpherbig) [inline data/base-filters.yaml]: ❓ Could this be merged with line 416?

File: data/base-filters.yaml (hunk 2 above)
Comment (jandersson) [reply data/base-filters.yaml]: The full message is something like `Trabe Chalice (Shattered, 28 minutes)`

File: data/base-filters.yaml (hunk 1 above)
Comment (jandersson) [reply data/base-filters.yaml]: Negative, there is more text

File: data/base-filters.yaml (hunk 3 above)
Comment (jandersson) [reply data/base-filters.yaml]: Yea

File: data/base-filters.yaml (hunk 2 above)
Comment (rpherbig) [reply data/base-filters.yaml]: This is fine then

**Comment by rpherbig** [comment]
It looks like there is a merge conflict that needs resolved before I can merge this.

**Pull request merged**